server.go

```diff
@@ -680,6 +682,8 @@ func (server *Server) loadConfig(configurations configs, globalConfiguration Glo
 						}
 
 						var negroni = negroni.New()
+						metricsMiddlewareBackend := middlewares.NewMetricsWrapper(middlewares.NewPrometheus(frontend.Backend))
```

Comment:
We should make it configurable (for the feature), which specific "exporter" to use. WDYT @enxebre?